projects/aca-shared/rules/src/navigation.rules.spec.ts

```diff
@@ -225,6 +225,38 @@ describe('navigation.evaluators', () => {
     });
   });
 
+  describe('isDetails', () => {
+    it('should return [true] if url ends with `/details`', () => {
```

Comment:
It returns the 'boolean' type. Please make it 'true' instead of [true]. Please change it in all other places in this file.

Reply:
Done!